bin/docker-build.js

```diff
@@ -57,9 +57,10 @@ process.on('message', function (message) {
     assert.optionalNumber(message.timeoutSeconds, 'timeoutSeconds');
 
     // Setup log streams.
+    var commandType = message.payload.command;  // Either 'build' or 'commit'
```

Comment:
Should `commandType` be tested for being either `'build'` or `'commit'` and error if anything else?